process/eventsInterceptor.go

```diff
@@ -39,10 +41,20 @@ func (ei *eventsInterceptor) ProcessBlockEvents(eventsData *data.ArgsSaveBlockDa
 
 	events := ei.getLogEventsFromTransactionsPool(eventsData.TransactionsPool.Logs)
 
+	txs := make(map[string]transaction.Transaction)
+	for hash, tx := range eventsData.TransactionsPool.Txs {
+		txs[hash] = tx.Transaction
```

Comment:
so currently you ignore the execution order?
will this come in a future PR? as we will require to expose this as well.

Reply:
yes, it will come later on, after we have the node version in line